02-Reproject-Raster-In-R.Rmd

```diff
@@ -42,6 +43,12 @@ After completing this activity, you will:
 
 * Be able to reproject a raster in R
 
+###Challenge Code
+Throughout the lesson we have Challenges that reinforce learned skills. Possible
```

Comment:
Shorter Text Option:

NEON Data lessons include challenges that reinforce learned skills. Solutions for each challenge, can be found in the `R` code, available for download on the bottom of each lesson page.